src/main/java/core/basesyntax/service/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.operations.Operation;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FileReaderServiceImpl implements FileReaderService {
+    @Override
+    public List<FruitTransaction> read(File file) {
```

Comment:
```suggestion
    public List<FruitTransaction> read(String filePath) {
```

It's better to pass the path to the file rather than the file itself.